lib/string-checker.js

```diff
@@ -93,27 +93,22 @@ StringChecker.prototype = {
     checkString: function(source, filename) {
         filename = filename || 'input';
 
-        var sourceTree;
-        var parseError;
-
-        try {
-            sourceTree = JsFile.parse(source, this._esprima, this._configuration.getEsprimaOptions());
-        } catch (e) {
-            parseError = e;
-        }
-
-        var file = this._createJsFileInstance(filename, source, sourceTree);
```

Comment:
Are you trying to remove all usage of this? Why wouldn't you just update it to use the new object format and then use it within this function to generate `file` (as it used to) and remove your local duplicate: https://github.com/jscs-dev/node-jscs/pull/1433/files?diff=unified#diff-c4ec64ef009f51578bddc596b5b6ddc2R225?

A little confused here.